packages/nx/release/changelog-renderer/index.ts

```diff
@@ -184,13 +184,6 @@ export default class DefaultChangelogRenderer {
     }
 
     if (this.isVersionPlans) {
-      this.conventionalCommitsConfig = {
```

Comment:
I am honestly not sure why this was hardcoded to defaults, it means custom config isn't respected even when present. It definitely makes sense to get rid of this, the defaults will have been set in `config.ts` in the case the user is not customizing